app/services/data_request_update_service.rb

```diff
@@ -48,7 +48,7 @@ def log_message
       # i18n-tasks-use t('cases.data_requests.update.log_message_pages_changed')
       I18n.t(".log_message_pages_changed",
              request_type: I18n.t("helpers.label.data_request.request_type.#{data_request.request_type}"),
-             location: @data_request.location,
+             location: @data_request.data_request_area.decorate.location,
```

Comment:
The object passed here should be decorated, and data_request has a decorated location method, so this shouldn't need changing.
```suggestion
             location: @data_request.location,
```